From e806bce7428b2358ef49d58859cdcfab93634e0b Mon Sep 17 00:00:00 2001
From: Xie Haocheng <haocheng.xie@amd.com>
Date: Fri, 4 Nov 2022 17:03:47 +0800
Subject: [PATCH] defconfig: add CONFIG_AMD_HSMP=m to openeuler_defconfig
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: e806bce7428b2358ef49d58859cdcfab93634e0b
Modified-by-SEL: No


amd inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5YNPT
CVE: NA

--------------------------------

Enable AMD Host System Management Port feauture built as a moudle
in x86 openeuler defconfig.

Signed-off-by: Xie Haocheng <haocheng.xie@amd.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 228a657e4fa8..9b5f792cb62e 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -6482,6 +6482,7 @@ CONFIG_MXM_WMI=m
 CONFIG_ACERHDF=m
 # CONFIG_ACER_WIRELESS is not set
 CONFIG_ACER_WMI=m
+CONFIG_AMD_HSMP=m
 CONFIG_APPLE_GMUX=m
 CONFIG_ASUS_LAPTOP=m
 # CONFIG_ASUS_WIRELESS is not set
-- 
2.33.0

